packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.styles.ts

```diff
@@ -72,6 +72,7 @@ export const getStyles = (props: IContextualMenuStyleProps): IContextualMenuStyl
         paddingTop: '5px',
         backgroundColor: palette.neutralLight
       }
-    ]
+    ],
+    subComponentStyles: { callout: {} }
   };
```

Comment:
I'm confused by this.. should this not do something here to customize the callout?

Reply:
Not as default styles. This was added to enable styling the ContextualMenu's callout from Fluent theme. The default styles didn't change so I am leaving it empty.